gradle/versions.gradle

```diff
@@ -134,7 +134,7 @@ dependencyManagement {
     dependency 'com.azure:azure-core-http-netty:1.13.5'
 
     dependency 'com.zaxxer:HikariCP:5.0.1'
-    dependency 'org.postgresql:postgresql:42.5.3'
+    dependency 'org.postgresql:postgresql:42.6.0'
```

Comment:
Was this fix an issue? Probably worthy of a changelog entry if that's the case

Reply:
This was originally to fix an issue reported by owasp dependency-check plugin, however, it was later turned out to be false positive i.e. PostgreSQL 12 is affected, not the JDBC driver. I am going to revert this back to `42.5.3`. The JDBC driver should be update from a separate PR.